domains/grid/platform-parser/warpcastParser.ts

```diff
@@ -0,0 +1,19 @@
+export const PLATFORM_PARSING_PARAMETERS_WARPCAST: PlatformParsingParameters = {
+  regexWithCallbacks: [
+    // Match whole embed code
+    {
+      regex: createIframeRegex('https:\\/\\/warpcast\\.com\\/(?<id>[\\w.-]+)'),
```

Comment:
Ideally it should be ```console
- 16 characters or fewer
- Latin alphabet, numbers or dash (`-`)
- Cannot start with a dash
- Must end in (`.eth`), i.e. ENS-compatible names like [`[cb.id](http://cb.id/)`](http://cb.id) aren’t supported at this time
- Not supported at this time:
    - subdomains, e.g. `abc.test.eth`
    - DNS to ENS
```
So maybe something like `\\/(?<id>[a-z0-9][a-z0-9-.]{0,15}$)`. They don't actually support unicode or other chars. Only a-z0-9 and - it seems. Although the "." in the name is not in the offficial docs although they say that ENS names work and they obviously require a dot. I suppose invalid names would just end up with an error page and should not be that bad.